server/src/main/java/org/elasticsearch/action/search/SearchTransportService.java

```diff
@@ -382,7 +384,7 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
-    public static void registerRequestHandler(TransportService transportService, SearchService searchService) {
+    public void registerRequestHandler(TransportService transportService, SearchService searchService) {
```

Comment:
Proposal: shall we instrument all of the handlers registered in this class, excluding the proxy ones? That would mean instrumenting clear scroll and free context too? I think that would not hurt.

Reply:
Can do, added as part of https://github.com/elastic/elasticsearch/pull/102557/commits/76ed555a39d08000fcb27e914efe3257d2888b76